googlesearch/__init__.py

```diff
@@ -72,3 +72,7 @@ def search(term, num_results=10, lang="en", proxy=None, advanced=False, sleep_in
                     else:
                         yield link["href"]
         sleep(sleep_interval)
+
+        if start == 0:
+            print("No result found with this search term", escaped_term)
```

Comment:
Please return an empty list or throw an error instead, unintended logging can be annoying

Reply:
Thanks for checking. Updated